lib/account_deleter.rb

```diff
@@ -69,10 +68,6 @@ def delete_standard_person_associations
     end
   end
 
-  def disconnect_contacts
```

Comment:
And the tests ensures that `contacts` are empty after the `AccountDeleter` run, so that shouldn't be a problem when something changes in the future.